src/app/sensors/details/sensor-detail.html

```diff
@@ -10,24 +10,49 @@
         <label for="sensor-detail__id">Device name</label>
         <input required [(ngModel)]="sensor.mqttId"
                ngControl="mqttId"
-               type="text" id="sensor-detail__id" placeholder="Enter descriptive name"/>
+               #mqttId="ngForm"
+               type="text" id="sensor-detail__id"
+               placeholder="Enter descriptive name"
+               required/>
+        <div [hidden]="!mqttId.touched || mqttId.valid" class="alert">
+            MQTT ID should be filled
+        </div>
     </div>
     <div class="sensor-detail__type">
         <label for="sensor-detail__id">Type</label>
         <input [(ngModel)]="sensor.type"
                ngControl="type"
-               type="text" id="sensor-detail__type" placeholder="Enter type"/>
+               #type="ngForm"
+               type="text" id="sensor-detail__type"
```

Comment:
if you've started splitting each attribute into a new line -- do that for id too